StackExchange.Redis/StackExchange/Redis/RedisChannel.cs

```diff
@@ -38,6 +39,7 @@ private RedisChannel(byte[] value, bool isPatternBased)
         {
             Value = value;
             IsPatternBased = isPatternBased;
+            IsKeyspaceChannel = value != null && Encoding.UTF8.GetString(value).ToLower().StartsWith("__key");
```

Comment:
Instead of creating a `byte[]` for each comparison - it'd be far cheaper to pre-compute the byte array of `"__key"` once in a static and compare them instead :)

Reply:
Here is the fixed version.
https://pastebin.com/Q9T0Q99K